### DISCUSSION TITLE
Zuordnung von:  sensor.pvfassade_energy

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#2253] [Q&A]
Hallo,

ich habe eine openDTU Fusion mit HM-800 und HMS-1000 aktiv, zusätzlich noch Home Assistant (HA) und MQTT installiert und greife darüber auf die Daten zu.

Im HA gibt es einen Sensor:  "sensor.pvfassade_energy".
Weiß jemand, wo der herkommt (ich vermute aus der openDTU?) und was die 3 Werte sum, change und state genau bedeuten?

Vielen Dank im Voraus
Werner

**Comment by Tpaydt** [comment]
Der Name des Sensors sieht nicht nach OpenDTU aus.
Geh auf dem Sensor in HA (3 Punkte oben rechts)schau was unter Zusatzinformationen angezeigt wird, dort sieht man woher die Daten kommen.

Zeig der Sensor überhaupt irgendwelche Werte an?

**Comment by Fokker-1** [comment]
Hi,

vielen Dank für die Hinweise.

Aber leider habe ich "... Geh auf dem Sensor in HA (3 Punkte oben rechts) schau was unter Zusatzinformationen ..." nicht in HA gefunden. Bin da neu, erst einige Tage aktiv.

Wäre schön, wenn Du noch genauere Infos schreiben könntest, wie der Weg dahin ist.

Vorab schon mal vielen Dank!

Grüße
Werner

**Comment by Tpaydt** [comment]
![Sensor HA](https://github.com/user-attachments/assets/2390ebe0-8516-4995-8294-b00e635326b4)
Dein Sensor wird dir ja in der Übersicht angezeigt. Diesen anklicken dann wird dieser angezeigt wie auf dem Bild, dort dann die auf die 3 Punkte oben rechts.

**Comment by Fokker-1** [comment]
Hallo,

 noch einmal vielen Dank für die Unterstützung, super.

Das Problem konnte gestern Abend geklärt werden.
Das ist ein zum Aufsetzen und Testen des HA temporär verwendter Sensor, der nicht zum Original-System gehört und dann gestern Abend auch gleich gelöscht wurde. Bei den Aktionen hatte mich ein Bekannter remote unterstützt, ich bin da ja "Frischling" ---

Viel Grüße
Werner
